### ISSUE TITLE
Nothing happens if another app is set as always on VPN 

### ISSUE THREAD

**Issue by codl** [pacien/tincapp#89]
When another app is designated as the always-on VPN, regardless of whether or not that app actually has the VPN running, trying to connect to a network in tinc does nothing. It stays on the start screen and nothing is shown. I figure it may be an android limitation that only the current always-on app can set up a VPN but it would be nice to at least show an error.

**Comment by pacien** [comment]
It should already be reporting an error in this situation.
Could you post the app's logs?

**Comment by codl** [comment]
Sure thing. I cleared out my logs, killed the app, and tried again, and the only thing it logged is:

```
23:42:00.043 [main] INFO  org.pacien.tincapp.context.App - Starting tinc app 0.27 (release build), running on REL (8.0.0)
```

**Comment by pacien** [comment]
Fixed with commit 3bc69093654f61c723a70bca9d57ffd57062cb2f, in v0.28.
